test/test_browser.py

```diff
@@ -2929,7 +2929,7 @@ def test_glfw3(self, args):
 
   @requires_graphics_hardware
   def test_glfw_events(self):
```

Comment:
Can you use `@parameterized` to run the test both with and without proxy-to-worker?

By the way are you actually using `-sPROXY_TO_WORKER` in your project in production?  Or are you managing the worker yourself?

Reply:
Do you have an open issue for this?   Can we link to it from the PR description here?

Reply:
Yes, I've tagged it in https://github.com/emscripten-core/emscripten/issues/19912 as well.